nomad/state/events_state_store.go

```diff
@@ -0,0 +1,47 @@
+package state
```

Comment:
Keep this in the same file. We should break it up at some point into a bit higher level groups: [node, eval, alloc, etc]